db/migrate/20250725060536_remove_action_value_from_ticket_logs.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RemoveActionValueFromTicketLogs < ActiveRecord::Migration[7.2]
+  def change
+    remove_column :ticket_logs, :action_value, :string
+
+    reversible do |dir|
+      dir.down do
+        TicketLogChange.find_each do |ticket_log_change|
+          ticket_log_change.ticket_log.update!(action_value: ticket_log_change.field_value) if ticket_log_change.status?
```

Comment:
`if ticket_log_change.status?` is the same argument as above: The enum should probably be `prefix: true` or `prefix: :field`, depending on your taste.